src/Scripting/VM/VirtualMachine.cpp

```
@@ -0,0 +1,1281 @@
+					ins.opcode, 2 // 2 hex chars = 1 byte
+				);
+			}
+		}
+		catch (const VMRuntimeException &) {
+			// Propagate exception
+			throw;
+		}
+		catch (const VMFinishedExecTick &) {
+			throw;
+		}
+		catch (const std::exception &e) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)